applications/dashboard/models/class.mediamodel.php

```diff
@@ -204,6 +207,16 @@ private function normalizeAndValidate(array $row): array {
      * @inheritdoc
      */
     public function saveUploadedFile(UploadedFile $file, array $extraArgs = []): array {
+        if (in_array('maxImageHeight', $extraArgs)) {
```

Comment:
This appears to be a bug. `in_array` tests values, not keys. The PHP best practice for options arrays is this:
```php
$extraArgs += [
    'maxImageHeight' => self::NO_IMAGE_DIMENSIONS_LIMIT,
   'maxImageWidth' => self::NO_IMAGE_DIMENSIONS_LIMIT,
];
```

Then you can make your checks to change it below.